### ISSUE TITLE
run "python setup.py bdist_wheel" error

### ISSUE THREAD

**Issue by aoke79** [intel-analytics/ipex-llm#11419]
Dear,
I'm trying to follow the steps to install npu extension by "https://github.com/intel-analytics/ipex-llm/tree/main/python/llm/example/NPU/HF-Transformers-AutoModels/Model/llama2", when I run "python setup.py bdist_wheel" it shows below, please have a look at:

-- Configuring done (4.6s)
-- Generating done (0.2s)
-- Build files have been written to: C:/AIGC/npu/intel-npu-acceleration-library/build/temp.win-amd64-cpython-310/Release cmake --build . --config Release
MSBuild version 17.9.8+b34f75857 for .NET Framework

  Checking Build System
  Building Custom Rule C:/AIGC/npu/intel-npu-acceleration-library/CMakeLists.txt
  intel_npu_acceleration_library.vcxproj -> C:\AIGC\npu\intel-npu-acceleration-library\build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\lib\Release\intel_npu_acceleration_library.dll
  Building Custom Rule C:/AIGC/npu/intel-npu-acceleration-library/CMakeLists.txt
building 'intel_npu_acceleration_library' extension
error: unknown file type '.txt' (from 'CMakeLists.txt')

**Comment by MeouSker77** [comment]
sorry, I can't reproduce this error.Could you check whether your cmake version is >= 3.12 and

**Comment by MeouSker77** [comment]
A successful build should has output like this:
```
(yishuo) arda@XIAOXIN02 D:\yishuo\npu-library>python setup.py bdist_wheel
C:\Users\arda\miniforge3\envs\yishuo\lib\site-packages\setuptools\dist.py:320: InformationOnly: Normalizing 'v1.2.0' to '1.2.0'
  self.metadata.version = self._normalize_version(self.metadata.version)
running bdist_wheel
running build
running build_py
creating build
creating build\lib.win-amd64-cpython-310
creating build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
copying intel_npu_acceleration_library\compiler.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
copying intel_npu_acceleration_library\device.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
copying intel_npu_acceleration_library\dtypes.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
copying intel_npu_acceleration_library\modelling.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
copying intel_npu_acceleration_library\optimizations.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
copying intel_npu_acceleration_library\quantization.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
copying intel_npu_acceleration_library\_version.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
copying intel_npu_acceleration_library\__init__.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library
creating build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\base.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\bindings.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\compression.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\convolution.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\factory.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\linear.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\matmul.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\mlp.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\ops.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\qlinear.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\qmatmul.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\runtime.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\sdpa.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\tensor.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\utils.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
copying intel_npu_acceleration_library\backend\__init__.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend
creating build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\nn
copying intel_npu_acceleration_library\nn\autograd.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\nn
copying intel_npu_acceleration_library\nn\conv.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\nn
copying intel_npu_acceleration_library\nn\functional.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\nn
copying intel_npu_acceleration_library\nn\linear.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\nn
copying intel_npu_acceleration_library\nn\llm.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\nn
copying intel_npu_acceleration_library\nn\module.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\nn
copying intel_npu_acceleration_library\nn\__init__.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\nn
creating build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\functional
copying intel_npu_acceleration_library\functional\scaled_dot_product_attention.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\functional
copying intel_npu_acceleration_library\functional\__init__.py -> build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\functional
running build_ext
cmake D:\yishuo\npu-library -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=D:\yishuo\npu-library\build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\lib -DCMAKE_BUILD_TYPE=Release -DSETUPTOOL_BUILD=True      
-- Building for: Visual Studio 17 2022
-- Selecting Windows SDK version 10.0.22621.0 to target Windows 10.0.22631.
-- The C compiler identification is MSVC 19.38.33133.0
-- The CXX compiler identification is MSVC 19.38.33133.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.38.33130/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.38.33130/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Setting Visual Studio specific flags for the entire build
-- OpenVINO library URL: https://storage.openvinotoolkit.org/repositories/openvino/packages/nightly/2024.3.0-15502-66093834e38/w_openvino_toolkit_windows_2024.3.0.dev20240524_x86_64.zip
CMake Warning (dev) at C:/Users/arda/miniforge3/envs/yishuo/Library/share/cmake-3.29/Modules/FetchContent.cmake:1352 (message):
  The DOWNLOAD_EXTRACT_TIMESTAMP option was not given and policy CMP0135 is
  not set.  The policy's OLD behavior will be used.  When using a URL
  download, the timestamps of extracted files should preferably be that of
  the time of extraction, otherwise code that depends on the extracted
  contents might not be rebuilt if the URL changes.  The OLD behavior
  preserves the timestamps from the archive instead, but this is usually not
  what you want.  Update your project to the NEW behavior or specify the
  DOWNLOAD_EXTRACT_TIMESTAMP option with a value of true to avoid this
  robustness issue.
Call Stack (most recent call first):
  CMakeLists.txt:81 (FetchContent_Declare)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE
-- Configuring done (23.5s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_BUILD_TYPE


-- Build files have been written to: D:/yishuo/npu-library/build/temp.win-amd64-cpython-310/Release
cmake --build . --config Release
适用于 .NET Framework MSBuild 版本 17.8.3+195e7f5a3

  1>Checking Build System
  Building Custom Rule D:/yishuo/npu-library/CMakeLists.txt
cl : 命令行  warning D9002: 忽略未知选项“/arch:SSE2” [D:\yishuo\npu-library\build\temp.win-amd64-cpython-310\Release\intel_npu_acceleration_library.vcxproj]
  bindings.cpp
    正在创建库 D:/yishuo/npu-library/build/temp.win-amd64-cpython-310/Release/Release/intel_npu_acceleration_library.lib 和对象 D:/yishuo/npu-library/build/temp.win-amd64-cpython-310/Release/Release/intel_npu_acc
  eleration_library.exp
  intel_npu_acceleration_library.vcxproj -> D:\yishuo\npu-library\build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\lib\Release\intel_npu_acceleration_library.dll
  Building Custom Rule D:/yishuo/npu-library/CMakeLists.txt
C:\Users\arda\miniforge3\envs\yishuo\lib\site-packages\setuptools\_distutils\cmd.py:66: SetuptoolsDeprecationWarning: setup.py install is deprecated.
!!

        ********************************************************************************
        Please avoid running ``setup.py`` directly.
        Instead, use pypa/build, pypa/installer or other
        standards-based tools.

        See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html for details.
        ********************************************************************************

!!
  self.initialize_options()
installing to build\bdist.win-amd64\wheel
running install
running install_lib
creating build\bdist.win-amd64
creating build\bdist.win-amd64\wheel
creating build\bdist.win-amd64\wheel\intel_npu_acceleration_library
creating build\bdist.win-amd64\wheel\intel_npu_acceleration_library\backend
copying build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend\base.py -> build\bdist.win-amd64\wheel\.\intel_npu_acceleration_library\backend
copying build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend\bindings.py -> build\bdist.win-amd64\wheel\.\intel_npu_acceleration_library\backend
copying build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend\compression.py -> build\bdist.win-amd64\wheel\.\intel_npu_acceleration_library\backend
copying build\lib.win-amd64-cpython-310\intel_npu_acceleration_library\backend\convolution.py -> build\bdist.win-amd64\wheel\.\intel_npu_acceleration_library\backend
...
```

**Comment by MeouSker77** [comment]
close this issue because intel-npu-acceleration-library has released version 1.3, which contains int8 and int4 fix, so no need to compile it from source code

#11446
